worker/src/direct_invocation/ws_direct_server_runner.rs

```diff
@@ -17,35 +17,32 @@
 
 use crate::direct_invocation::ws_handler::CreateWsHandler;
 use log::*;
-use sgx_types::{sgx_enclave_id_t, sgx_status_t};
 use std::sync::Arc;
 use std::thread;
 use std::thread::JoinHandle;
+use substratee_enclave_api::enclave_api::EnclaveApi;
 use ws::{listen, Handler};
 
-// TODO replace these extern C e-calls with a EnclaveAPI member field in the server impl
-extern "C" {
-    fn initialize_pool(eid: sgx_enclave_id_t, retval: *mut sgx_status_t) -> sgx_status_t;
-}
-
```

Comment:
Now we don't need to make the e-call directly anymore, but rather use the enclave API trait